### PR TITLE
Job.isCreated

### PR DESCRIPTION
**Pull request by vogdb**

### CODE REVIEW

**Comment by vogdb** [comment]
@njam or @tomaszdurka please review

**Comment by vogdb** [comment]
The best thing in the morning is a cup of coffee and this PR @njam @tomaszdurka ;-)

**Comment by vogdb** [comment]
@fvovan please review

File: src/job.js [hunk 1]
```diff
@@ -37,6 +37,13 @@ Job.prototype.isRunning = function() {
   return this.data.status == PulsarJob.STATUS.RUNNING;
 };
 
+/**
+ * @returns {boolean}
+ */
+Job.prototype.isCreated = function() {
+  return this.data && this.data.status == PulsarJob.STATUS.CREATED;
```
Comment (fvovan) [inline src/job.js]: Is `this.data` checking really needed? It's always defined at line 18, isn't it?

Comment (vogdb) [inline src/job.js]: `this.data` is set only when job was successfully created on pulsar-rest-api server. Until then it is `null`. https://github.com/cargomedia/pulsar-rest-api-client-node/blob/master/src/client.js#L39

Comment (tomaszdurka) [inline src/job.js]: Ah I would kind of agree with @fvovan, but haven't touched js for some time :)

Otherwise lgtm.

**Comment by vogdb** [comment]
@fvovan please review

**Pull request merged**